web-app/packages/app/package.json

```diff
@@ -19,13 +19,11 @@
     "i18n:report": "vue-cli-service i18n:report --src './src/admin/**/*.?(js|vue)' --locales './src/admin/locales/**/*.json'"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-free": "^6.4.2",
     "@gtm-support/vue2-gtm": "^1.0.0",
```

Comment:
do we need this?

Reply:
yes :( in new design, we will clean it to tabler icons only :100:

Reply:
sorry, I meant GTM